src/utils/makeReactNativeConfig.js

```diff
@@ -130,10 +129,11 @@ const getDefaultConfig = ({
       }),
       /**
        * By default, sourcemaps are only generated with *.js files
-       * We need to use the plugin to configure *.bundle to emit sourcemap
+       * We need to use the plugin to configure *.bundle (Android, iOS - development)
+       * and *.jsbundle (iOS - production) to emit sourcemap
        */
       new webpack.SourceMapDevToolPlugin({
-        test: /\.(js|css|bundle)($|\?)/i,
+        test: /\.(js|css|(js)bundle)($|\?)/i,
```

Comment:
Why in this regex `(js)` is not optional? Like in
https://github.com/callstack/haul/pull/289/files#diff-332bac151e40e6cd74cea97441a4144bR158

Reply:
Good catch, thanks. It had to accidentally miss the character.